src/test/java/no/unit/nva/model/PublicationTest.java

```diff
@@ -31,4 +43,49 @@ protected Object produceFramedPublication(JsonNode publicationWithContext) throw
         options.setPruneBlankNodeIdentifiers(true);
         return JsonLdProcessor.frame(input, frame, options);
     }
+
+    @Test
+    public void updatingDoiRequestStatusForValidTransitionChangeReturnsUpdatedDoiRequestStatus()
+        throws InvalidIssnException, MalformedContributorException {
+        var publication = PublicationGenerator.generateJournalArticlePublication();
+        publication.updateDoiRequestStatus(REJECTED);
```

Comment:
parametrize the test.

Reply:
The code looks like you are testing for separate cases. I did not understand that you were testing for REJECTED->APPROVED. Can you add a comment on each line?

```
publication.updateDoiRequestStatus(REJECTED); //initial state
        assertThat(publication.getDoiRequest().getStatus(), is(equalTo(REJECTED)));

        publication.updateDoiRequestStatus(APPROVED); // next state
        assertThat(publication.getDoiRequest().getStatus(), is(equalTo(APPROVED)));
```